upgrade/upd_2.5.11-to-2.7.0/index.php

```
@@ -235,48 +283,413 @@ public function apply_addsessioncookieprefs()
+     * Clear compiled Smarty templates and module caches.
+     *
+     * Uses SystemMaintenance::CleanCache() with folder IDs:
+     *   1 = compiled templates, 2 = xoops_cache, 3 = Smarty cache
+     *
+     * @return bool true on success
+     */
+    public function apply_cleancache(): bool
+    {
+        require_once XOOPS_ROOT_PATH . '/modules/system/class/maintenance.php';
+        try {
+            $maintenance = new \SystemMaintenance();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '625', column '32'). (undefined)

(MissingImport)